baip-0007.md

```diff
@@ -0,0 +1,58 @@
+BAIP: 0007
+Title: Proposal on bitASSETS reward and BTS destruction
+Authors: cn-vote   zhouxiaobao-2010
+Status: Accepted
+Type: Consensus
+Created: 2020-06-01
+
+# Abstract
+The purpose of this proposal is to establish a scientific financial model of BTS, make reasonable use of the commission income of the committee, open up resources and reduce expenditure, and promote the stable growth of BTS system value.
+
+# Specification
+
+### bitASSETS reward
+
+Intelligent assets are the original assets of the system, and the corresponding holders should enjoy the dividend of system growth.
+Reward amount: 20000 BTS per day
+Participating assets: bitCNY, bitUSD, bitEUR, bitRUBLE
+
+1.Reward conditions:
+1)The holding amount should exceed X. For example: those over 1000 yuan can participate in reward.
+2)The holding amount needs to last for a period of time . For example, 7 days. Rewards can only be distributed if the holding amount meets 1000bitcny or above for 7 consecutive days.
+
+2.Reward method:
+1)Participating accounts should register at the beginning of each cycle.Account does not participate in reward if not registered. The registration method can be to transfer any small number of bitassets to the specified account.
+2)The reward needs to be collected actively. If the reward is not collected within the time limit, this part of the reward will not be accumulated.The collection method can be to transfer any small number of bitasset to the specified account.
```

Comment:
Too unclear, describe in details how exactly rewards collection works.

Reply:
In this paragraph, the proposal puts forward the standard requirements on how to implement the reward for the organizer. The organizer will separately announce the ways to realize the reward. We have already consulted, and it can be realized.

About that, the organizers set up a statistical account. Transfer any small amount of bitasset from the account applying for lock in reward to this statistical account is deemed as the registered account. The organizer will check the bitasset of the account participating in the lock in reward every day and calculate the amount of reward that the account should receive.

When the lockup period expires, the same operation shall be taken, and the account participating in the lockup reward transfers any small amount of bitasset to the statistical account, then the organizer will distribute the reward to the target account.

Reply:
I didn't meant I need additional details, I just suggesting to improve the text of this BAIP on lines I commented on (this is how code review is supposed to work). My comment is for 

>2)The reward needs to be collected actively

Reply:
I'm sorry, I don't quite understand what you mean. Do you mean to include all these details?

Reply:
Yes, improve original text.